groupnews-service/src/test/java/com/epam/internship/GroupServiceTest.java

```diff
@@ -89,4 +102,52 @@ public void shouldFailWhenDescCharacterNumberExceeds2000() {
 		when(userRepository.findOne(USER_ID)).thenReturn(new UserEntity());
 		systemUnderTest.createGoup(USER_ID, GROUP_TITLE, RandomStringUtils.random(2001));
 	}
+
+	@Test
+	public void shouldReturnListOfGroupsBelongingToUser() {
+		// Given
+		UserEntity userEntity = UserEntity.builder().id(USER_ID).build();
+		GroupEntity groupEntity1 = GroupEntity.builder().id(GROUP_ID_1).title(GROUP_TITLE)
+				.description(GROUP_DESCRIPTION).createdBy(userEntity).build();
+		GroupEntity groupEntity2 = GroupEntity.builder().id(GROUP_ID_2).title(GROUP_TITLE_2)
+				.description(GROUP_DESCRIPTION_2).createdBy(userEntity).build();
+		User user = User.builder().id(USER_ID).build();
+		Group group1 = Group.builder().id(GROUP_ID_1).title(GROUP_TITLE).description(GROUP_DESCRIPTION).createdBy(user)
+				.build();
+		Group group2 = Group.builder().id(GROUP_ID_2).title(GROUP_TITLE_2).description(GROUP_DESCRIPTION_2)
+				.createdBy(user).build();
+		List<Group> expectedGroups = new ArrayList<>();
+		expectedGroups.addAll(Arrays.asList(group1, group2));
```

Comment:
You can init expectedGroups with `List<Group> expectedGroups = Arrays.asList(group1, group2);`